src/sentry/utils/json.py

```diff
@@ -70,6 +70,7 @@ def dump(value, fp, **kwargs):
 
 
 def dumps(value, escape=False, **kwargs):
+    # Legacy use. Do not use. Use dumps_htmlsafe
```

Comment:
What's wrong with this? Isn't it the same as `dumps_htmlsafe` except for the `mark_safe`?